src/mainwindow.h

```diff
@@ -75,13 +76,18 @@ private slots:
 
     void on_actionWEST_triggered();
 
+    void setNumberOfSteps(QString& c);
+
 private:
 	Ui::MainWindow* ui;
 	GuiInteractiveController guiInteractiveController;
 	friend class GuiInteractiveController;
 
 	SvgViewToolchain svgViewToolchain;
 
+    QLabel* labelNumberOfSteps;
```

Comment:
Whenever you have class member variables that are *raw pointers* like here,
you have to make sure, that they become deleted in the class's destructor.
(You can omit delete in destructor and set a parent instead which takes ownership of the pointer. But be careful since it is easy to produce memory leaks here.)
Also make sure that you always keep a sane pointer there, i.e. either `nullptr` or a pointer to a valid object.

Reply:
obsolete since we pass ownership to a parent